docs/changelog.md

```diff
@@ -3,7 +3,7 @@
 All notable changes to this project will be documented in this file.
 
 ## 1.2.0
-* Alerting functionality implemented for Glue Data Quality events
+* Added Glue Data Quality service (alerts, metrics, digest, optional Grafana dashboard).
```

Comment:
let's maybe rephrase a bit. to something like "Added integration with AWS Glue Data Quality (..."